examples/cov2d_simulation.py

```diff
@@ -54,20 +55,19 @@
 filters = [RadialCTFFilter(pixel_size, voltage, defocus=d, Cs=2.0, alpha=0.1)
            for d in np.linspace(defocus_min, defocus_max, defocus_ct)]
 
-# Create a simulation object with specified filters
+# Create a simulation object with specified filters and 3D map of a 70S Ribosome
+logger.info('Load 3D map and creat simulation object.')
 sim = Simulation(
     n=num_imgs,
+    vols_file=os.path.join(DATA_DIR, 'clean70SRibosome_vol_65p.mrc'),
     C=num_maps,
     filters=filters
 )
 
-# Load 3D map from the data file, corresponding to the experimentally obtained EM map of a 70S Ribosome.
-logger.info('Load 3D map and downsample to desired grids.')
-vols = np.load(os.path.join(DATA_DIR, 'clean70SRibosome_vol.npy'))
-vols = vols[..., np.newaxis]
-
-# Downsample the 3D map to 8X8X8
-vols = downsample(vols, (img_size*np.ones(3, dtype=int)))
+# Downsample the 3D map to desired resolution This can be done by the internal function of sim object.
+# Below we use alternative implementation to obtain the exact result with Matlab version.
+logger.info(f'Downsample 3D map to desired grids of {img_size} x {img_size} x {img_size}.')
+vols = downsample(sim.vols, (img_size*np.ones(3, dtype=int)))
 sim.vols = vols
```

Comment:
What's the point of loading them inside `Simulation` if we're going to change them afterwards?

Reply:
The simulation class does not have the downsample function for 3D yet. we need to implement it later. The function I used can generate downsampled volume exactly the same as Matlab code for comparison. In future it can be rewritten to use the function from simulation class.

Reply:
A notice has been created in issue #61 for future implementation.